sms-sending/config.yml

```diff
@@ -0,0 +1,45 @@
+---
+title: Auto Send SMS
+description: Auto send SMS after a call gets disconnected using Platform API.
+categories:
+- Architect
+steps:
+- title: Introduction
+  content: |
+    The main objective of this tutorial is to showcase how to create an interaction, and send SMS through a queue with Platform API. The given sample code will send a message after an agent drops off a call. This tutorial will also cover how to create a notification channel, subscribe to a topic and listen to a web socket.
+- title: Authenticate with PureCloud
+  content: |
+    First, authenticate the application with pureCloud.
+- title: Create a Notification Channel 
+  content: |
+    Create a notification channel and subscribe to the topic _{"id": "v2.routing.queues.{queueID}.conversations.calls"}_. This topic will listen to a queues incoming and outgoing calls. After setting up the notification channel, create a web socket object using the notification channel URI.
```

Comment:
Instead of underscores, use backticks for literal values: 
```
`{"id": "v2.routing.queues.{queueID}.conversations.calls"}`
```

Reply:
got it